### DISCUSSION TITLE
Get deployment job status when using deploy hook

### DISCUSSION THREAD

**Discussion by teeto** [vercel/community#165] [Idea]
When you have a deploy hook url and make a get request to the url it starts a deployment and returns a json string with the job id and status (pending). I need another url to request the job status by the id to see when the deployment is finished.
I would like to make a button on a cms that when clicked makes a get request to the deploy hook.
It would be great to show some indication to the user when the deploy is finished.
I think that having the job id i could query the status but how could i do it making a request to the api?
I have looked for the documentation but i got nothing.

**Comment by Pieparker** [comment]
Hey there, we've moved this to the Vercel Community Discussions as a feature suggestion. Thank you!

**Comment by ThomasMarcelo** [comment]
Any updates on this?

**Comment by bhaveyexpansionjs** [comment]
Any Update on this ???
